modules/gdnative/nativearvr/doc_classes/ARVRInterfaceGDNative.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="ARVRInterfaceGDNative" inherits="ARVRInterface" category="Core" version="3.0.alpha.custom_build">
```

Comment:
Did you check that the editor knows how to handle this file, or said otherly, was it generated as expected by doctool or did you write it yourself? I know that doctool supports `modules/<name>/doc_classes/` but I don't know if it handles `modules/<name>/<subfolder>/doc_classes/`.

Reply:
No I didn't checks this actually, I created that file when the this logic was still in its own module before Karroffel suggested I should just add it into the gdnative module.